setup.py

```diff
@@ -1,4 +1,11 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of Viper - https://github.com/viper-framework/viper
+# See the file 'LICENSE' for copying permission.
+from __future__ import unicode_literals
+from __future__ import absolute_import
```

Comment:
I don't want to add any code that would support Python 2.6, and absolute_import seems to do that: http://python-future.org/futurize.html?highlight=absolute_import#stage-1-safe-fixes
Are you sure it is useful?